### DISCUSSION TITLE
Save last used mods on each song

### DISCUSSION THREAD

**Discussion by rasputozen** [ppy/osu#15230] [Ideas] [answered]
I use custom playback speed and OD regularly for different songs. It would be nice if the last used mod settings on specific songs were recorded and auto-loaded whenever that song is played again.

**Comment by bdach** [comment, marked as answer]
Mod presets are already a part of an upcoming redesign of the mod selection overlay: https://www.figma.com/file/NkdREXr6wFkA99oLw8PnBv/Client%2FMod-Selection?node-id=0%3A1

**Reply by rasputozen** [reply]
Looks great, can't wait!
